atlp-rwanda/phantom-be-codebandits
# #CP-39: User update own profile

### What does this PR do?

Allow a user to edit their own profile from the frontend using their access_token to send that

### Description of Task to be completed

- [x] User can upload profile picture
- [x] User can change their number 

### How should this be manually tested?


```
git clone https://github.com/atlp-rwanda/phantom-be-codebandits
cd phantom-be-codebandits
git fetch origin
git checkout <your branch>
npm install
//configure the .env according to the .env.sample file
// start the database
npm run dev
// open browser or postman (Check linked public workplace for testing)
```

### Any background context you want to provide?

The picture uploads are handled with cloudinary. A cloudinary account is required to be able to upload the picture

### What are the relevant pivotal tracker stories?

[#CP-39

### Screenshots (if appropriate)
